.travis.yml

```diff
@@ -12,6 +12,7 @@ branches:
 env:
   global:
     - CACHE_NAME=JOB1
+    - MONGODB_VERSION=3.4.16
```

Comment:
Talking about version pinning, I believe what we have now `/mongodb-org/3.4`  in repo with `major.minor` would be overall a more preferable approach.
I think that in `mongodb` context pinning patch version instead of giving a better control would, in fact, work against us.